qemu/tests/cfg/balloon_in_use.cfg

```diff
@@ -0,0 +1,30 @@
+- balloon_in_use:
+    type = driver_in_use
+    only Windows
+    extra_params += "-device virtio-balloon-pci,id=ballooning"
+    iterations = 5
+    ratio = 0.5
```

Comment:
We need to consider windows 32 bit guests ,  as well 
if we start vm  w/ 8GB ,then balloon it to 4G on qemu side ,there should be no empty mem pages on windows guests.